applications/serial_lte_modem/src/main.c

```diff
@@ -76,7 +76,7 @@ static void on_modem_failure_shutdown(struct k_work *work)
 	int ret = nrf_modem_lib_shutdown();
 
 	ARG_UNUSED(work);
-	sprintf(rsp, "#XMODEM: SHUTDOWN,%d", ret);
+	sprintf(rsp, "\r\n#XMODEM: SHUTDOWN,%d\r\n", ret);
```

Comment:
Should this be added to the response in `nrf_modem_fault_handler` as well?

Reply:
Yes, you're quite right.